Products/GenericSetup/tests/test_context.py

```diff
@@ -91,27 +98,23 @@ def test_readDataFile_nonesuch(self):
 
     def test_readDataFile_simple(self):
 
-        from string import printable
-
         FILENAME = 'simple.txt'
-        self._makeFile(FILENAME, printable)
+        self._makeFile(FILENAME, printable_bytes)
```

Comment:
Wouldn't it be much easier to fix `_makeFile` such that it encoded values as needed?

Reply:
I guess I was figuring the "Python 3 way" is to avoid implicit encoding/decoding when possible. Not that that's applied consistently in this PR, and especially here in test code I guess it'd be fine.